app/controllers/api/v1/partner_requests_controller.rb

```diff
@@ -11,6 +11,7 @@ def create
     request = Request.new(request_params)
 
     if request.save
+      NotifyPartnerJob.perform_now(request.id)
```

Comment:
I think here should be perform_async, to execute when workers are available

Reply:
yeah, I thought the same, but I looked for examples in other jobs and mailers in this project, and they used `perform_now` and `deliver_now`. So I'm not sure what is preferred in this project.